holmes.go

```diff
@@ -722,6 +722,11 @@ func (h *Holmes) writeProfileDataToFile(data bytes.Buffer, dumpType configureTyp
 		h.Errorf("failed to write profile to file(%v), err: %s", fileName, err.Error())
 		return
 	}
+
+	if h.opts.DumpOptions.DumpGrToLogger {
```

Comment:
maybe `DumpToLogger` should be better, not only for `goroutine`.

Reply:
OKay, and I will add a comment to remind the user it will make huge log output if he enables the option.